README.md

```diff
@@ -1,44 +1,18 @@
-# edgedelta-lambda-extension
-Edge Delta lambda extension to monitor AWS lambda functions.
+# Introduction
```

Comment:
For the example log section, should we update to the latest log format @baturozgur or completely remove example log format?